### PR TITLE
git-setup: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
Thanks, @CleanMachine1!

**Review by navarroaxel — APPROVED** [review]

File: pages/common/git-setup.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# git setup
+
+> Setup a Git repository in a specific directory, add all files and make a commit with the added files.
```
Comment (marchersimon) [inline pages/common/git-setup.md:3]: How about that?
```suggestion
> Setup a Git repository in a specific directory and commit all files.
```
This would also apply to the two example descriptions.

File: pages/common/git-setup.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-setup>.
+
+- Make a Git repository in the current directory, add all files and make a commit with the added files:
```
Comment (marchersimon) [inline pages/common/git-setup.md:7]:
```suggestion
- Create a Git repository in the current directory, add all files and make a commit with the added files:
```
I think `create` is more appropriate here.

File: pages/common/git-setup.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+> Setup a Git repository in a specific directory and commit all files.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-setup>.
+
+- Create a Git repository in the current directory, add all files and make a commit with the added files:
+
+`git setup`
+
+- Make a Git repository in a specific directory, add all files and make a commit with the added files:
```
Comment (CleanMachine1) [inline pages/common/git-setup.md:11]:
```suggestion
- Create a Git repository in a specific directory, add all files and make a commit with the added files:
```

File: pages/common/git-setup.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# git setup
+
+> Setup a Git repository in a specific directory and commit all files.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-setup>.
+
+- Create a Git repository in the current directory, add all files and make a commit with the added files:
```
Comment (marchersimon) [inline pages/common/git-setup.md:7]: I think we should shorten this too
```suggestion
- Create a Git repository in the current directory and commit all files:
```

File: pages/common/git-setup.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+`git setup`
+
+- Create a Git repository in a specific directory, add all files and make a commit with the added files:
```
Comment (marchersimon) [inline pages/common/git-setup.md:11]:
```suggestion
- Create a Git repository in a specific directory and commit all files:
```

**Review by marchersimon — APPROVED** [review]

File: pages/common/git-setup.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# git setup
+
+> Setup a Git repository in a specific directory and commit all files.
```
Comment (CleanMachine1) [inline pages/common/git-setup.md:3]:
```suggestion
> Create a Git repository in a specific directory and commit all files.
```
